memo.rb

```diff
@@ -22,27 +22,44 @@ def h(text)
   end
 end
 
-def manipulate_db(manipulation, memo = nil)
-  conn = PG.connect(dbname: DB_NAME)
-  query = QUERIES[manipulation]
-  case manipulation
-  when :select_one then conn.exec_params(query, [memo[:id]]).first
-  when :select_all then conn.exec_params(query)
-  when :insert then conn.exec_params(query, [memo[:id], memo[:title], memo[:text]])
-  when :update then conn.exec_params(query, [memo[:title], memo[:text], memo[:id]])
-  when :delete then conn.exec_params(query, [memo[:id]])
-  end
+def connect_db
+  PG.connect(dbname: DB_NAME)
+end
+
+def select_one(id)
+  conn = connect_db
+  conn.exec_params(QUERIES[:select_one], [id]).first
```

Comment:
```suggestion
  conn.exec_params('SELECT * FROM memo WHERE id = $1', [id]).first
```

みたいにベタ書きした方が可読性が高いと思いますー